### PR TITLE
development ui dice roller

### PR DESCRIPTION
**Pull request by briansetzesglobant**
Implementation of the graphical user interface of the dice roller with six dice

image

### CODE REVIEW

File: lib/DiceRollerUI.dart [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        crossAxisSpacing: Constants.bodyCrossAxisSpacing,
+        children: List.generate(
+          Constants.bodyChildren,
+          (int index) {
+            return Container(
+              decoration: BoxDecoration(
+                borderRadius: BorderRadius.circular(
+                  Constants.bodyBorderRadius,
+                ),
+                //color: Colors.green,
```
Comment (FranciscoGaraventa) [inline lib/DiceRollerUI.dart]: Remove this line. Try to not add comment code when you do a PR

File: lib/utils/Constants.dart [hunk 2]
```diff
@@ -0,0 +1,9 @@
+abstract class Constants {
+  static const int bodyCrossAxisCount = 2;
+  static const double bodyPadding = 25.0;
+  static const double bodyMainAxisSpacing = 15.0;
+  static const double bodyCrossAxisSpacing = 40.0;
+  static const int bodyChildren = 6;
+  static const double bodyBorderRadius = 25.0;
+  static const double bottomNavigationBarChildren = 25.0;
+}
```
Comment (FranciscoGaraventa) [inline lib/utils/Constants.dart]: Remember to do CTRL+ALT+L. Here is missing a newline at end of file

File: lib/utils/TextStyles.dart [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      189,
+      56,
+      27,
+      150.0,
+    ),
+    fontSize: 25.0,
+    fontWeight: FontWeight.normal,
+    height: 2.0,
+  );
+}
```
Comment (FranciscoGaraventa) [inline lib/utils/TextStyles.dart]: Here is like the previous comment. Do CTRL+ALT+L.

File: lib/utils/TextStyles.dart [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+      fontSize: 25.0,
+      fontWeight: FontWeight.normal,
+      height: 2.0,
+    );
+  static const TextStyle bottomNavigationBarValue =TextStyle(
+    color: Color.fromRGBO(
+      189,
+      56,
+      27,
+      150.0,
```
Comment (FranciscoGaraventa) [inline lib/utils/TextStyles.dart]: Add this to constants file, and fontSize, height

File: lib/DiceRollerUI.dart [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(widget.title),
+      ),
+      backgroundColor: Color.fromARGB(
+        255,
+        0,
+        217,
+        255,
```
Comment (FranciscoGaraventa) [inline lib/DiceRollerUI.dart]: Add this to constant file

File: lib/DiceRollerUI.dart [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+  @override
+  _DiceRollerUIState createState() => _DiceRollerUIState();
+}
+
+class _DiceRollerUIState extends State<DiceRollerUI> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(widget.title),
```
Comment (FranciscoGaraventa) [inline lib/DiceRollerUI.dart]: You missed a trailing comma here

File: lib/dice_roller_ui.dart [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+      appBar: AppBar(
+        title: Text(
+          widget.title,
+        ),
+      ),
+      backgroundColor: Color.fromARGB(
+        Constants.backgroundColor255,
+        Constants.backgroundColor0,
+        Constants.backgroundColor217,
+        Constants.backgroundColor255,
```
Comment (FranciscoGaraventa) [inline lib/dice_roller_ui.dart]: Try to not add to the name of the constant the value that is going to have. In this case you can change this names with: backgroundColorAlpha, backgroundColorRed, backgroundColorGreen, backgroundColorBlue

File: lib/utils/constants.dart [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+  static const double bodyMainAxisSpacing = 15.0;
+  static const double bodyCrossAxisSpacing = 40.0;
+  static const int bodyChildren = 6;
+  static const double bodyBorderRadius = 25.0;
+  static const double bottomNavigationBarChildren = 25.0;
+
+  static const int TextStyle189 = 189;
+  static const int TextStyle56 = 56;
+  static const int TextStyle27 = 27;
+  static const double TextStyle150_0 = 2;
```
Comment (FranciscoGaraventa) [inline lib/utils/constants.dart]: As i mentioned above, don't use the value that the constant will take as part of constant name.

Comment (FranciscoGaraventa) [reply lib/utils/constants.dart]: Remember that we use lowerCamelCase to name constants

File: lib/utils/text_styles.dart [hunk 9]
```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/material.dart';
+import 'constants.dart';
+
+abstract class TextStyles {
+  static const TextStyle bottomNavigationBarText = TextStyle(
+    fontSize: 28.0,
+    fontWeight: FontWeight.normal,
+    height: 2.0,
```
Comment (FranciscoGaraventa) [inline lib/utils/text_styles.dart]: Add fontSize and height to constant file.

**Review by FranciscoGaraventa — COMMENTED** [review]
Change some camel case name of constants

**Review by MarcosCGlobant — APPROVED** [review]
LGTM

**Review by FranciscoGaraventa — APPROVED** [review]
LGTM

**Pull request merged**